.github/CODEOWNERS

```diff
@@ -69,6 +69,24 @@
 #CC# /x-pack/plugins/drilldowns/ @elastic/kibana-app-services
 #CC# /packages/kbn-interpreter/ @elastic/kibana-app-services
 
+### Observability Plugins
+
+# Observability Shared
+/x-pack/plugins/observability/ @elastic/observability-ui
+
+# Logs and Metrics (Infra)
+/x-pack/plugins/infra/ @elastic/logs-metrics-ui
+/x-pack/plugins/infra/server/routes/log* @elastic/logs-metrics-ui @elastic/logs-ui
+/x-pack/plugins/infra/public/pages/logs/ @elastic/logs-metrics-ui @elastic/logs-ui
+/x-pack/plugins/infra/server/routes/metrics* @elastic/logs-metrics-ui @elastic/metrics-ui
+/x-pack/plugins/infra/public/pages/metrics/ @elastic/logs-metrics-ui @elastic/metrics-ui
```

Comment:
I added a few subdirectories with more specific team call-outs in preparation for one day separating this out a little better and making it easier to tease these apart. Everything will fall back to the top line here if not specifically mentioned, and it all still includes the logs-metrics-ui team anyway, for notifications, etc.